---
title: 500: INTERNAL_SERVER_ERROR
url: https://docs.attio.com/sdk/introduction/graphql
crawled_at: unknown
depth: 2
parent_url: https://docs.attio.com/sdk/introduction
---

# 500: INTERNAL_SERVER_ERROR

**Source:** [https://docs.attio.com/sdk/introduction/graphql](https://docs.attio.com/sdk/introduction/graphql)

**500** : INTERNAL_SERVER_ERROR Code: `INTERNAL_UNEXPECTED_ERROR` ID: `cle1::h7dvs-1748991212536-2aad0be057c2`
